boards/arm/mec172xevb_assy6906/mec172xevb_assy6906_defconfig

```diff
@@ -20,3 +20,5 @@ CONFIG_I2C=y
 CONFIG_ESPI=y
 CONFIG_SPI=y
 CONFIG_SPI_ASYNC=y
+CONFIG_PWM=y
+CONFIG_PWM_XEC=y
```

Comment:
PWM is not a minimal peripheral, so it should not be in the board defconfig per porting guidelines (actually same applies for some other entries in that file).